src/libLocalDescriptorCache/LocalDescDb.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+//import "google/protobuf/descriptor.proto";
+
+
+package localDescDb;
+
+// Array of hosts
+message DescriptorDb{
+    repeated Host hosts = 1;
+    string gwhisper_version = 2;
+    //TODO: gwhisper-Version
+}
+
+//evtl hier Dsc Message definieren und un ServerDb message verwenden, Muss ich service definieren, der Descriptoren holt?
+
+message Host{
+    string hostAddress = 1;
+    google.protobuf.Timestamp lastUpdate = 2;
+    repeated bytes file_descriptor_proto = 3;
+    repeated string serviceList = 4;
```

Comment:
field names in ptoro file should be `snake_case` according to the google guidelines